packages/office-ui-fabric-react/src/components/Slider/docs/new/SliderOverview.md

```diff
@@ -0,0 +1,5 @@
+A Slider provides a visual indication of adjustable content, as well as the current setting in the total range of content. Use a slider when you want people to set defined values (such as volume or brightness), or when people would benefit from instant feedback on the effect of setting changes.
```

Comment:
Can we add something about the axis and min/max range to clarify the parts that make up the slider?